hnn_core/tests/test_network.py

```diff
@@ -13,8 +14,19 @@ def test_network():
     params_fname = op.join(hnn_core_root, 'param', 'default.json')
     params = read_params(params_fname)
     net = Network(deepcopy(params))
+    neuron_network = _neuron_network(params)  # needed to populate net.cells
     for p in params:
         assert params[p] == net.params[p]
     assert len(params) == len(net.params)
     print(net)
-    print(net.cells[:2])
+    print(neuron_network.cells[:2])
+
+
+def test_plots():
+    hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+    net = Network(params)
+    simulate_dipole(net)
+    net.plot_input()
+    net.plot_spikes()
```

Comment:
I think we need to add CircleCI too. I can take care of this in another PR.

Reply:
I haven't used CircleCI, would like to see how it works. Thanks.